plugin-server/src/main/ingestion-queues/session-recording-v2/sessions/session-batch-recorder.ts

```diff
@@ -59,6 +64,7 @@ export class SessionBatchRecorder {
     private readonly partitionSizes = new Map<number, number>()
     private _size: number = 0
     private readonly batchId: string
+    public createdAt: number
```

Comment:
suggestion: As mentioned in another comment, this is not used within the batch recorder, we can remove it.